system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/DiscoverySystemTest.java

```diff
@@ -25,6 +28,23 @@ public void testNodeDiscoveryRest() throws InterruptedException {
 
         ElasticsearchNodesResponse nodesResponse = new ElasticsearchNodesResponse(tasks, NODE_COUNT);
         assertTrue("Elasticsearch nodes did not discover each other within 5 minutes", nodesResponse.isDiscoverySuccessful());
-    }
+
```

Comment:
Shouldn't you clean out the data from the host machine after finishing the tests?